<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeMachine</class>
 <widget class="QDialog" name="ChangeMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>205</y>
     <width>100</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>214</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>430</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>id станка: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Состояние (Рабочий = 0, В ремонте = 1, Сломан = 2): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_state"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Состояние ремня (от 0 до 65535): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_belt_wear"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Состояние двигателя (от 0 до 65535): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_engine_wear"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Количество оборотов (от 0 до 32000): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="_speed_engine"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
